core/src/main/scala/cats/instances/sortedSet.scala

```diff
@@ -94,7 +94,7 @@ private[instances] trait SortedSetInstancesBinCompat0 {
 private[instances] trait SortedSetInstancesBinCompat1 extends LowPrioritySortedSetInstancesBinCompat1 {
   // TODO: Remove when this is no longer necessary for binary compatibility.
   implicit override def catsKernelStdHashForSortedSet[A: Order: Hash]: Hash[SortedSet[A]] =
-    cats.kernel.instances.sortedSet.catsKernelStdHashForSortedSet[A]
+    cats.kernel.instances.sortedSet.catsKernelStdHashForSortedSet1[A]
```

Comment:
@travisbrown 
Just noticed this, should this method be deprecated and un-implicit like the one above?  The `Todo` comment should probably be removed too?

Reply:
@kailuowang What's happening here is tricky: these instances were in the wrong place (`cats.instances.SortedSetInstances1` instead of `cats.kernel.instances`), and now that they're in the right place these bincompat traits get the methods from both, which results in an `inherits conflicting members` compiler error, which means we have to bubble overrides all the way to the top.